test_regress/t/t_aggregate_dtype_compare.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
Please rename tests to t_lint_dtype_compare (or t_math_dtype_compare), the first word should match an existing group of tests.